client/client-multi/client-java-core/src/main/java/org/kaaproject/kaa/client/channel/impl/channels/DefaultOperationsChannel.java

```diff
@@ -290,7 +291,7 @@ public synchronized void setMultiplexer(KaaDataMultiplexer multiplexer) {
     }
 
     @Override
-    public synchronized void setServer(ServerInfo server) {
+    public synchronized void setServer(TransportConnectionInfo server) {
```

Comment:
I will add TODO with link to the following issue: http://jira.kaaproject.org/browse/KAA-126